radix-engine-interface/src/constants.rs

```diff
@@ -11,284 +14,303 @@ use crate::model::*;
-    113,
-    220,
-    243,
-    95,
-    55,
-    167,
-    67,
-    74,
-    9,
-    105
+    37,
+    227,
+    231,
+    245,
+    106,
+    85,
+    104,
+    249,
+    221,
+    70,
+    50,
+    6,
+    109,
+    237,
+    77,
+    32,
+    128
 );
 
 /// The address of the faucet component, test network only.
 pub const FAUCET_COMPONENT: ComponentAddress = construct_address!(
     EntityType::NormalComponent,
-    87,
-    220,
-    4,
-    44,
-    216,
-    203,
-    145,
-    111,
-    54,
-    48,
-    2,
-    10,
-    31,
-    51,
-    124,
-    236,
-    90,
-    84,
-    207,
-    239,
-    164,
-    197,
-    8,
-    79,
-    190,
-    60
+    241,
+    80,
+    14,
+    193,
+    40,
+    120,
+    1,
+    16,
+    50,
+    105,
+    249,
+    218,
+    195,
+    64,
+    201,
+    162,
+    23,
+    173,
+    172,
+    153,
+    29,
+    117,
+    113,
+    45,
+    245,
+    16
 );
 
 pub const EPOCH_MANAGER: SystemAddress = construct_address!(
     EntityType::EpochManager,
-    242,
-    112,
-    114,
-    176,
-    201,
-    24,
-    36,
-    161,
-    165,
-    168,
-    98,
-    35,
-    142,
-    88,
     111,
-    226,
-    199,
-    205,
-    55,
+    216,
+    129,
+    144,
+    194,
+    100,
+    251,
+    131,
+    110,
+    208,
+    77,
     97,
-    235,
-    46,
-    52,
+    44,
+    107,
+    166,
+    93,
+    138,
+    28,
+    57,
+    240,
+    45,
     60,
-    218,
-    190
+    200,
+    116,
+    131,
+    121
 );
 
 pub const CLOCK: SystemAddress = construct_address!(
     EntityType::Clock,
-    198,
-    192,
-    61,
     210,
     4,
-    230,
-    44,
-    57,
+    203,
+    199,
+    253,
+    87,
+    86,
+    55,
+    225,
+    160,
+    209,
+    125,
+    34,
+    246,
+    206,
+    141,
+    224,
+    160,
+    236,
+    54,
     219,
-    60,
-    174,
-    35,
-    57,
-    88,
-    91,
-    98,
-    186,
-    244,
-    0,
-    251,
-    251,
-    77,
-    116,
-    187,
-    229,
-    39
+    221,
+    233,
+    10,
+    33,
+    79
 );
 
 /// The ED25519 virtual resource address.
 pub const EDDSA_ED25519_TOKEN: ResourceAddress = construct_address!(
     EntityType::Resource,
-    15,
-    142,
-    146,
-    10,
-    167,
-    159,
-    83,
-    52,
-    157,
-    10,
+    100,
+    122,
+    90,
     153,
-    116,
-    110,
-    23,
-    181,
-    146,
-    65,
-    189,
-    81,
-    225,
-    154,
-    187,
-    80,
-    173,
-    107,
-    106
+    192,
+    230,
+    68,
+    232,
+    52,
+    111,
+    194,
+    67,
+    139,
+    246,
+    24,
+    111,
+    166,
+    139,
+    122,
+    227,
+    235,
+    71,
+    163,
+    178,
+    99,
+    94
 );
 
 pub const EPOCH_MANAGER_BLUEPRINT: &str = "EpochManager";
 pub const CLOCK_BLUEPRINT: &str = "Clock";
 pub const RESOURCE_MANAGER_BLUEPRINT: &str = "ResourceManager";
 pub const PACKAGE_BLUEPRINT: &str = "Package";
 pub const TRANSACTION_PROCESSOR_BLUEPRINT: &str = "TransactionProcessor";
+
+pub struct AuthAddresses;
```

Comment:
I would prefer having these constants closer to their module rather than type (ie should be under Auth rather than constants) so that we can keep things modular (e.g. compile a kernel with no auth enabled).

Reply:
Not sure how practical this is, but will move it closer to auth module.